sdks/python/apache_beam/runners/portability/universal_local_runner.py

```diff
@@ -83,6 +86,16 @@ def cleanup(self):
       time.sleep(0.1)
     self._subprocess = None
 
+  @staticmethod
+  def default_docker_image():
+    if 'USER' in os.environ:
+      # Perhaps also test if this was built?
+      logging.info('Using latest built Python SDK docker image.')
+      return os.environ['USER'] + '-docker.apache.bintray.io/beam/python:latest'
+    else:
+      logging.warnign('Could not find a Python SDK docker image.')
```

Comment:
`warning`
(is this codepath tested / does it need to be?)

Reply:
Apparently it's tested somewhere because the presubmits are failing due to the `warnign` module not existing.

Reply:
USER is not set in the jenkins environment; when have enough for a full end-to-end test we may need to pass the appropriate path manually (which is probably better anyway...)